src/libraries/Microsoft.PowerFx.Core/Functions/Blob/UriBlob.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    public class UriBlob : BlobElementBase
```

Comment:
>UriBlob [](http://example.com/codeflow?start=17&length=7)

Since our partners don't need a URI-based blobs for now, can we just not have that in this first iteration? And add later if needed?

Reply:
I'll remove this class